string tf_prefix
int32 state
int32 pwm1
int32 pwm2